docs/shadow_service.md

```diff
@@ -0,0 +1,107 @@
+# Shadow Service User Manual
+
```

Comment:
Need to describe what's the Shadow service, including:
- What're the values the shadow service can bring to users?
- What're the scenarios the shadow service the user should be used?
- What're the main features of the shadow service.

Reply:
Thanks for the comments, have updated accordingly.